diff --git a/CMakeLists.txt b/CMakeLists.txt
index 55352cc..2798256 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -796,7 +796,6 @@ if(SDL2MIXER_INSTALL)
         COMPONENT devel
     )
 
-    if(SDL2MIXER_BUILD_SHARED_LIBS)
         # Only create a .pc file for a shared SDL2_mixer
         set(VERSION ${FULL_VERSION})
         set(SDL_VERSION ${SDL_REQUIRED_VERSION})
@@ -826,7 +825,6 @@ if(SDL2MIXER_INSTALL)
             file(INSTALL DESTINATION \"\${CMAKE_INSTALL_PREFIX}/${PC_DESTDIR}\"
                 TYPE FILE
                 FILES \"${CMAKE_CURRENT_BINARY_DIR}/SDL2_mixer.pc\")" CONFIG Release COMPONENT devel)
-    endif()
 
     if(SDL2MIXER_BUILD_SHARED_LIBS AND (APPLE OR (UNIX AND NOT ANDROID)))
         install(
